site/docs/v1/tech/shared/_deleting-your-account.adoc

```diff
@@ -0,0 +1,25 @@
+To delete your account, take the following steps, in this order:
+
+1. https://account.fusionauth.io/[Log into your account].
+2. Delete all deployments.
+3. Switch your account to the community plan.
+4. Delete your company.
+
+[NOTE.note]
+====
+FusionAuth requires that customers delete their own deployments using the account portal.
+
+FusionAuth is **not able** to cancel accounts with deployments that could potentially contain critical user data.
+====
+
+To remove your deployments, please https://account.fusionauth.io/account/deployment/[log into your account and navigate to your deployments]. Once you see your deployment, you can click the [uielement]#Action# button and select [uielement]#Delete#. After a deployment is destroyed, you won't be charged going forward. Any previous usage outside of a free trial will be billed.
+
+Then, https://account.fusionauth.io/account/plan/[switch your plan to to the community plan]. After the current billing cycle is done, you won't be charged. (Plans run for a month.)
+
+After deleting your deployments and changing your plan, your account will not be charged. It is inactive and can be deleted if you desire.
+
+To delete the company, navigate to [breadcrumb]#Company -> Danger Zone# and follow the instructions there.
+
+image::admin-guide/delete-account.png[Deleting your account.,width=1200,role=bottom-cropped]
+
+If you cannot delete the company using this method, you may have an account type which requires FusionAuth to delete your account. This occurs if you are invoiced, for example. In this case, after you have made your account inactive, please https://account.fusionauth.io/account/support/[open a support ticket to delete your account].
```

Comment:
> If you cannot delete the company using this method, you may have an account type which requires FusionAuth to delete your account. This occurs if you are invoiced, for example. In this case, after you have made your account inactive, please https://account.fusionauth.io/account/support/[open a support ticket to delete your account].

If you cannot delete the company using this method, you may have an account type that requires FusionAuth to delete your account (this occurs if you are invoiced, for example). In this case, please confer with your Account Team for further instructions.  A support ticket may also be required to remove your company.